### PR TITLE
Update Multiselect Widget Documentation

### PR DESCRIPTION
**Pull request by serhiyzhovnir**
##  Purpose of this pull request

This pull request (PR) updates Multiselect Widget Documentation. 

Updates:
* Added the information about `nextPageUrl` option 
* Added the Methods section

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget-multiselect.html
- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget-multiselect.html

## Links to Magento source code

- https://github.com/magento/magento2/blob/2.2/app/code/Magento/Tax/Controller/Adminhtml/Rule/AjaxLoadRates.php
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Tax/Controller/Adminhtml/Rule/AjaxLoadRates.php
- https://github.com/magento/magento2/blob/2.3/lib/web/mage/multiselect.js
- https://github.com/magento/magento2/blob/2.2/lib/web/mage/multiselect.js

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/javascript-dev-guide/widgets/widget-multiselect.md [hunk 1]
```diff
@@ -124,6 +125,112 @@ The search field update delay.
...
+
+```javascript
+$('#multiselect').multiselect2('isOptionsLoaded', [{"value": "2", "label": "Label #2"}]);
+```
+
+This method returns `true` if multiple selection items are loaded, and it returns `false` if they are not loaded. 
+
+### `clearMultiselectOptions()`
+
+The method clears all multi-select items.
```
Comment (rogyar) [inline guides/v2.2/javascript-dev-guide/widgets/widget-multiselect.md]: Could you clarify this point, please? The "clears" means deselect items or remove them? 
Thank you

Comment (serhiyzhovnir) [reply guides/v2.2/javascript-dev-guide/widgets/widget-multiselect.md]: The `clearMultiselectOptions()` method removes all multi-select items. The description of this method already adjusted. Thank you!

**Comment by serhiyzhovnir** [comment]
Hey @dobooth 
This PR is ready for your review.
Let me know if I need to add some additional modifications.
Thank you!

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Comment by meker12** [comment]
@dobooth Please add whatsnew, thx.

**Pull request merged**